.github/workflows/run-tests.yml

```
@@ -8,8 +8,11 @@ on:
     branches:
       - master
 
+permissions:
+  content: read
```

actionlint (1.7.4)

unknown permission scope "content". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)